packages/ckit/src/tx-builders/pw/TransferSudtPwBuilder.ts

```diff
@@ -4,75 +4,233 @@ import { Amount, Builder, Cell, RawTransaction, Transaction } from '@lay2/pw-cor
 import { Pw } from '../../helpers/pw';
 import { CkitProvider } from '../../providers';
 import { boom } from '../../utils';
+import { byteLenOfCkbLiveCell, byteLenOfSudt } from '../builder-utils';
 import { AbstractPwSenderBuilder } from './AbstractPwSenderBuilder';
 
 interface TransferOptions {
   readonly recipient: Address;
   readonly sudt: CkbTypeScript;
-  readonly amount: HexNumber;
+  amount: HexNumber;
+  policy: 'findAcp' | 'createCell' | 'findOrCreate';
 }
 
 export class TransferSudtPwBuilder extends AbstractPwSenderBuilder {
-  constructor(private options: TransferOptions, provider: CkitProvider, private sender: Address) {
+  constructor(private options: TransferOptions[], provider: CkitProvider, private sender: Address) {
```

Comment:
to keep the interface as consistent as possible, maybe we should change options like this way

```ts
interface TransferOptions {
  recipients: TransferSudtRecipient[];
}

```

Reply:
fixed